src/spec.c

```diff
@@ -651,6 +662,7 @@ void IndexSpecCache_Decref(IndexSpecCache *c) {
   }
   for (size_t ii = 0; ii < c->nfields; ++ii) {
     rm_free(c->fields[ii].name);
+    rm_free(c->fields[ii].path);
```

Comment:
Should not free if `path` == `name`.

Reply:
I need to do it for both allocation and free of the SpecCache.